framework/include/timeintegrators/TimeIntegrator.h

```diff
@@ -196,6 +202,9 @@ class TimeIntegrator : public MooseObject, public Restartable
   /// Boolean flag that is set to true if lumped mass matrix is used
   bool _is_lumped;
 
+  /// Boolean flag that is set to true if direct acceleration calculation is used
+  bool _is_direct;
+
```

Comment:
```suggestion
```

move to `CentralDifference`

Reply:
let's leave this here then.

Reply:
> let's leave this here then.

Should I ignore the below suggestion to move `_is_direct` into central difference?

Reply:
Yeah, and simply return by value here `virtual bool isDirect() const { return false; }`

Reply:
[36cf76e](https://github.com/idaholab/moose/pull/27991/commits/36cf76e60b00c7e3e4641fd082f78c4a580d7e63)

Let me know if this is what you had in mind